milestoneupdater/config.json

```diff
@@ -31,22 +31,13 @@
     "user_saml"
   ],
   "dueDates": {
-    "Nextcloud 12.0.12": "2018-10-11",
-    "Nextcloud 13.0.7": "2018-10-11",
-    "Nextcloud 14.0.1": "2018-10-11",
-    "Nextcloud 15": "2018-12-06"
+    "Nextcloud 14.0.1": "2018-09-25"
   },
   "closeMilestones": [
-    "Nextcloud 12.0.10",
-    "Nextcloud 13.0.6"
+    "Nextcloud 14"
   ],
   "addMilestones": [
-    "Nextcloud 12.0.12",
-    "Nextcloud 13.0.7",
-    "Nextcloud 13.0.8",
-    "Nextcloud 14",
-    "Nextcloud 14.0.1",
-    "Nextcloud 15"
+    "Nextcloud 14.0.2"
```

Comment:
Same here. There should be all currently open milestones in here. The script automatically checks if they are already present.